app/views/start_page/show.html.erb

```diff
@@ -11,5 +11,9 @@
       <li>get support from the Department for Education (DfE) when you apply</li>
       <li>tailor your application to suit different providers</li>
     </ul>
+
+    <%= link_to personal_details_path, role: 'button', draggable: false, class: 'govuk-button govuk-button--start' do %>
```

Comment:
```suggestion
    <%= link_to personal_details_path, role: 'button', class: 'govuk-button govuk-button--start' do %>
```

'cuse the nitpick!

Reply:
my bad — thought I'd 🔥 that one... one moment please...

Reply:
Ah, I see. There are two of them. They are now both in the bin.